guardian/manage-frontend
# Label payment method SVGs

## What does this change?

The SVG images used to indicate payment method are only accessible to sighted users. A screenreader user won't know that the payment method is, for example, a Visa card.

This PR adds an `aria-label` attribute around the SVGs to label the payment method.

## How to test

Using a screen reader navigate to the payment method section - the payment method should be read out. 

Using inspect element the SVGs or surrounding div should have an aria-label containing the payment type.

## Accessibility
